agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/httpclient/RedirectPolicy.java

```diff
@@ -43,7 +43,7 @@
   private static final Logger logger = LoggerFactory.getLogger(RedirectPolicy.class);
   public static final String INSTRUMENTATION_KEY = "instrumentationKey";
 
-  private final Cache<URL, String> redirectMappings = Cache.builder().setMaximumSize(100).build();
+  private final Cache<URL, String> redirectMappings = Cache.bounded(100);
```

Comment:
typo: bound instead of bounded? 
i think it's "cache bound". That's up to opentelemetry sdk to update that.

Reply:
I think `bounded` is ok also, e.g. "it's a bounded cache"